signing_service/signing_service/signing_service.py

```diff
@@ -159,13 +190,42 @@ def _parse_arguments() -> argparse.Namespace:
         type=int,
         help=f'Port on which Concent cluster is listening (default: {SIGNING_SERVICE_DEFAULT_PORT}).',
     )
-    parser.add_argument(
+
+    ethereum_private_key_parser_group = parser.add_mutually_exclusive_group()
+    ethereum_private_key_parser_group.add_argument(
+        '--ethereum-private-key',
+        dest='ethereum_private_key',
+        action=secret_provider_factory(read_command_line=True, base64_convert=True),
+    )
+    ethereum_private_key_parser_group.add_argument(
+        '--ethereum-private-key-path',
+        dest='ethereum_private_key',
+        action=secret_provider_factory(use_file=True, base64_convert=True),
+    )
+    ethereum_private_key_parser_group.add_argument(
+        '--ethereum-private-key-from-env',
+        dest='ethereum_private_key',
+        action=secret_provider_factory(env_variable_name='ETHEREUM_PRIVATE_KEY', base64_convert=True),
+    )
+    parser.set_defaults(ethereum_private_key=b64decode(os.environ.get('ETHEREUM_PRIVATE_KEY')) if os.environ.get('ETHEREUM_PRIVATE_KEY') is not None else None)
+
+    sentry_dsn_parser_group = parser.add_mutually_exclusive_group()
+    sentry_dsn_parser_group.add_argument(
         '--sentry-dsn',
-        default='',
         dest='sentry_dsn',
-        help=f'Sentry DSN for error reporting.',
```

Comment:
Why are you removing the description? It sounds useful. You should have it for every option.

Reply:
Restored